pandas/tests/io/excel/test_readers.py

```diff
@@ -67,6 +60,24 @@ def engine(request):
     return request.param
 
 
+@pytest.mark.parametrize("kwargs", [dict(), dict(engine=None)])
+def test_default_engine(datapath, monkeypatch, kwargs):
+    """
+    A test for default ExcelFile engine value (which is None)
+    """
+    monkeypatch.chdir(datapath("io", "data"))
+    expected = "xlrd"
+
+    result = pd.ExcelFile("blank.xls", **kwargs)
+    assert result.engine == expected
```

Comment:
This works for `ExcelFile` - can you think of a way to test this for `pd.read_excel` as well?

Reply:
So this test does testing implicitly for the default value of the engine parameter (which is `None`) for `pd.read_excel` as well, because under the hood `pd.read_csv` either creates an instance of the `ExcelFile` class and passes the engine parameter value to it or is provided with an `ExcelFile` instance through the `io` parameter:
```
    if not isinstance(io, ExcelFile):
        io = ExcelFile(io, engine=engine)
    elif engine and engine != io.engine:
        raise ValueError(
            "Engine should not be specified when passing "
            "an ExcelFile - ExcelFile already has the engine set"
        )
```

Reply:
Yea I just think that's fragile to always assume that's how this implementation will work - we'd lose coverage and expose ourselves to bugs if this gets refactored

Reply:
Yes, I agree with you. I'll try to figure out a way to test `read_excel` as well, but I'm afraid it's not gonna be pretty...